packages/shared/src/components/squads/SquadsToShare.tsx

```diff
@@ -63,7 +63,6 @@ export function SquadsToShare({
       onClick={() =>
         openNewSquadModal({
           origin: Origin.Share,
-          redirectAfterCreate: false,
```

Comment:
We now have the separate page so we should always redirect, alternatively we would have to remember the post id to get user back to it for sharing. Edge case?

Reply:
No, but the hook is also really specifically called modal, I'll think of the way to update it.

Reply:
Refactored here 91333732c13665fe09ef49056ce8d64255124d38 @sshanzel